GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -332,6 +332,14 @@ public override bool TryGetGVFSEnlistmentRoot(string directory, out string enlis
             return WindowsPlatform.TryGetGVFSEnlistmentRootImplementation(directory, out enlistmentRoot, out errorMessage);
         }
 
+        public override bool TryKillProcessTree(int processId, out int exitCode, out string error)
+        {
+            ProcessResult result = ProcessHelper.Run("taskkill", $"/pid {processId} /f /t");

```

Comment:
Do taskkill and pkill return success if the specified process ID is no longer running?

Reply:
They do not.  I can update them both to return success for those exit codes.   You agree that's worth changing?

Reply:
Oh actually I think I misread the old code.  I thought it was treating already shutting down as success, but now that I take another look at `TryKillRunningProcess` it was returning false in that scenario:

```
            catch (Win32Exception)	
            {	
                // Thrown when process is already terminating	
            }	
            catch (InvalidOperationException)	
            {	
                // Process already terminated	
            }	


            return false;
```

So I think we're OK

Reply:
Or maybe have the caller interpret the error code and report the process did not exist? (as opposed to a failure...)?

Reply:
Considering we send the exitCode / error code back, i think it's fine to fail here.  This is part of shut down so the only difference would be that extra logging.  If we need something fancier in the future we can definitely do that.